magenta-lib/src/main/scala/magenta/deployment_type/AmiCloudFormationParameter.scala

```diff
@@ -33,6 +33,11 @@ object AmiCloudFormationParameter extends DeploymentType {
   val amiParameter = Param[String]("amiParameter",
     documentation = "The CloudFormation parameter name for the AMI"
   ).default("AMI")
+  val amiParametersToTags = Param[Map[String, Map[String, String]]]("amiParametersToTags",
```

Comment:
Yes - actually we did start doing that, but were struggling to get the types aligning (for another reason). Perhaps we can now introduce `CfnParam` and `TagCriteria` (or similar) types.